tests/testthat/test-parse.r

```diff
@@ -0,0 +1,52 @@
+context("Parsing expressions")
+
+expect_equal(
+  parse_safe(c("", " ", " \n  \n ")),
+  expression(NA, NA, NA)
+)
+
+expect_equal(
+  parse_safe(c("A", "B", "C")),
+  expression(A, B, C)
+)
+
+expect_equal(
+  parse_safe(c("alpha", "beta", "gamma")),
+  expression(alpha, beta, gamma)
+)
+
+expect_equal(
+  parse_safe(c("alpha", "", "gamma", " ")),
+  expression(alpha, NA, gamma, NA)
+)
+
+expect_equal(
+  parse_safe(c("alpha ~ beta", " ", "integral(f(x) * dx, a, b)")),
+  expression(alpha ~ beta, NA, integral(f(x) * dx, a, b))
+)
+
+expect_equal(
+  parse_safe(c("alpha \n beta", " ", "integral(f(x) * dx, a, b)")),
+  expression(alpha, NA, integral(f(x) * dx, a, b))
+)
+
+expect_equal(
```

Comment:
This doesn't test anything new?

Reply:
You're right, some of the tests are probably redundant. I'll try to reduce redundancy.